### ISSUE TITLE
Flag to specify output format

### ISSUE THREAD

**Issue by TomWright** [TomWright/dasel#19]
**Is your feature request related to a problem? Please describe.**
It would be nice to be able to specify a desired output format.
This will allow you to easily transform data from one format to another.

**Describe the solution you'd like**
A new flag `-n`, `--out-parser` to specify the parser to use on output.
This should be optional. If it is not present then use the input parser.

```
echo '{"name":"Tom"}' | dasel -p json -n xml .
<name>Tom</name>
```

**Describe alternatives you've considered**
`-n` isn't a particularly declarative shorthand flag for this use-case. I'm open to suggestions on this.

**Additional context**
This new flag should be available on both `select` and `put` commands.

Providing a value of `-` should be treated the same as passing the `--plain` flag.

**Comment by slippycheeze** [comment]
Since I came here to suggest just this feature, for the argument question I suggest instead:
```
Flags:
  -r, --read FORMAT     Read the input in FORMAT.
  -w, --write FORMAT   Write the result in FORMAT.
  -p, --parser FORMAT   Shorthand for `-r FORMAT -w FORMAT`.
```

**Comment by TomWright** [comment]
Thanks for the suggestion! I think that works well

**Comment by TomWright** [comment]
@slippycheeze This is now implemented as of `v1.7.0`. Thanks for the suggestion!
